### PR TITLE
update accessibility statement

### PR DESCRIPTION
**Pull request by Donna-H**
## Changes in this PR:
update accessibility statement
## Jira card / Rollbar error (etc)
https://national-archives.atlassian.net/jira/software/projects/FCL/boards/136?selectedIssue=FCL-360
### Before
[before.webm](https://github.com/user-attachments/assets/07ecdd58-4e66-4dbe-b0b5-a0c30acef062)

### After
[after.webm](https://github.com/user-attachments/assets/b158c887-0f7c-448d-92b9-3bab58df8ebd)

- [ ] Requires env variable(s) to be updated

### CODE REVIEW

File: ds_judgements_public_ui/templates/pages/accessibility_statement.html [hunk 1]
```diff
@@ -105,17 +105,17 @@ <h2 id="section-contact">Contacting us or visiting in person</h2>
             chat. We provide information about
             <a href="https://www.nationalarchives.gov.uk/about/visit-us/">visiting in person</a>.
             People with health conditions or impairments that may impact their visit can
-            <a href="https://www.nationalarchives.gov.uk/about/visit-us/information-for-disabled-visitors/">find out more</a>.
+            <a href="https://www.nationalarchives.gov.uk/about/visit-us/information-for-disabled-visitors/">find out more about access building details</a>.
```
Comment (jacksonj04) [inline ds_judgements_public_ui/templates/pages/accessibility_statement.html:108]: The phrase "find out more about access building details" doesn't track - should it be "accessibility details"?

Comment (Donna-H) [reply ds_judgements_public_ui/templates/pages/accessibility_statement.html:108]: @Terry-Price please see @jacksonj04 comment images of before and after text changes
before
before-text
after
after-text

Comment (Terry-Price) [reply ds_judgements_public_ui/templates/pages/accessibility_statement.html:108]: Hi @Donna-H 
Do you mind changing the date at the top so it doesnt use 'th'.

So 'DD Month YYYY'

Comment (Donna-H) [reply ds_judgements_public_ui/templates/pages/accessibility_statement.html:108]: can we maybe change 'find out more about access building details' to ' find out more how we provide full access to all our public areas'  as the first line states on TNA website below ? @jacksonj04 @Terry-Price new-text

File: ds_judgements_public_ui/templates/pages/accessibility_statement.html [hunk 2]
```diff
@@ -41,7 +41,7 @@ <h3 class="anchor-links__header">On this page</h3>
       <div class="govuk-grid-column-two-thirds">
         <p class="date-time-info-page">
           <small>Last updated on
-            <time datetime="20-14-2024">20th August 2024</time>
+            <time datetime="03-10-2024">3 October 2024</time>
```
Comment (Donna-H) [inline ds_judgements_public_ui/templates/pages/accessibility_statement.html:44]: @Terry-Price the th has already been removed see above change in code and image below 
removed-th

File: ds_judgements_public_ui/templates/pages/accessibility_statement.html (hunk 1 above)
Comment (jacksonj04) [reply ds_judgements_public_ui/templates/pages/accessibility_statement.html:108]: Is it as straightforward as should be "building access details" not "access building details"? That would track!

Comment (Donna-H) [reply ds_judgements_public_ui/templates/pages/accessibility_statement.html:108]: @Terry-Price this short form can be retained then by switching order of text as @jacksonj04 suggested is that okay ?

File: ds_judgements_public_ui/templates/pages/accessibility_statement.html (hunk 2 above)
Comment (Terry-Price) [reply ds_judgements_public_ui/templates/pages/accessibility_statement.html:44]: Thanks @Donna-H

File: ds_judgements_public_ui/templates/pages/accessibility_statement.html [hunk 3]
```diff
@@ -105,7 +105,7 @@ <h2 id="section-contact">Contacting us or visiting in person</h2>
             chat. We provide information about
             <a href="https://www.nationalarchives.gov.uk/about/visit-us/">visiting in person</a>.
             People with health conditions or impairments that may impact their visit can
-            <a href="https://www.nationalarchives.gov.uk/about/visit-us/information-for-disabled-visitors/">find out more about access building details</a>.
+            <a href="https://www.nationalarchives.gov.uk/about/visit-us/information-for-disabled-visitors/">find out more about building access details</a>.
```
Comment (Donna-H) [inline ds_judgements_public_ui/templates/pages/accessibility_statement.html:108]: @Terry-Price switch text around as @jacksonj04 suggested

**Review by Terry-Price — APPROVED** [review]
All good, thanks @Donna-H

**Pull request merged**